### DISCUSSION TITLE
Table - Link Table Row to New Tab

### DISCUSSION THREAD

**Discussion by tailorck** [Budibase/budibase#5577] [Help]
I am using a table to display data for an application. I found that I can enable the rows to be clickable by selecting `link table rows` and then putting a link in the `link screen` option. I want the link to open in a new tab and see no option for that though. How can I do that?

**Comment by melohagan** [comment]
Hi @tailorck 

It does appear this option is missing, but the workaround is to add clickable links to the table rows:
![Screenshot 2022-04-26 at 11 32 55](https://user-images.githubusercontent.com/101575380/165281247-cfc5975c-4955-41cf-9131-50b33a445b89.png)

I've added a suggestion to the table improvements thread: https://github.com/Budibase/budibase/issues/3273
